apis/groups/v1alpha1/group_types.go

```diff
@@ -130,7 +130,8 @@ type GroupParameters struct {
 
 	// The parent group ID for creating nested group.
 	// +optional
-	ParentID *int `json:"parentId,omitempty"`
+	// +crossplane:generate:reference:type=github.com/crossplane-contrib/provider-gitlab/apis/groups/v1alpha1.Group
+	ParentID *string `json:"parentId,omitempty"`
```

Comment:
This is a breaking API change that we should better avoid. If dealing with `int`s is not possible with angryjet's code generate then the referencers need to be implemented manually where a conversion between int and string can be added.